flare_portal/experiments/forms.py

```diff
@@ -73,6 +73,39 @@ def save(self, *, experiment: Experiment) -> None:
 )
 
 
+class ParticipantDeleteForm(forms.Form):
+    participant: Participant = None
+    participant_id_confirm = forms.CharField(
+        max_length=25, label="Confirm Participant ID"
+    )
+
+    def __init__(self, participant, *args, **kwargs):
+        super(ParticipantDeleteForm, self).__init__(*args, **kwargs)
+        self.participant = participant
+
+    def clean(self):
+        super().clean()
```

Comment:
I think the more common pattern here is to use the return value of `clean()` instead of `self.cleaned_data`

```python
cleaned_data = super().clean()
```

Also don't forget to return `cleaned_data` from this method.

Reply:
Thanks!